recipe/boost_example.cpp

```diff
@@ -0,0 +1,11 @@
+#include <boost/archive/text_oarchive.hpp>
+#include <fstream>
+#include <string>
+
+int main()
+{
+  std::string filename = "output.txt";
+  std::ofstream ofs(filename.c_str());
+  boost::archive::text_oarchive oa(ofs);
+  return -1;
```

Comment:
IDK what this test is for, but any function that returns -1 will always fail. I think the return value here should be 0.

Reply:
```suggestion
  return 0;
```